packages/cli/src/lib/defaults/infra-modules/tezos/cmds.js

```diff
@@ -0,0 +1,7 @@
+const CMDS = {
```

Comment:
@namesty I'm confused by how these js modules get used, as I don't see them referenced in the docker-compose.yaml or Dockerfile.

Are they useful to users in some way? If so, how can they access them? Is this something common you think we should do (bundle helper scripts w/ the infra modules)?